opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -448,6 +446,13 @@ message Mapping {
   bool has_inline_frames = 9;
 }
 
+// A Stack represents a stack trace as a list of locations. The first location
```

Comment:
We should also add this to and update the helper ascii graph at the beginning of the file.